### ISSUE TITLE
Show Value Objects in action

### ISSUE THREAD

**Issue by ardalis** [ardalis/CleanArchitecture#302]
Add a value object property to a Project or ToDoItem in the initial domain model of the template to show how they're used.

see: #301

**Comment by MarkusGnigler** [comment]
Hi Steve, is there any suggestion for an example type for the value object?

Should there be only one example of it or should there also be actions operating on it?
How much of it should be covered in tests?

**Comment by danihengeveld** [comment]
This issue seems obsolete since there is a PhoneNumber ValueObject now.

**Comment by ardalis** [comment]
Agreed. And I'm working on some Vogen stuff in the /sample folder as well (not there yet but hopefully soon).
